docs/experimental-features.md

```diff
@@ -17,7 +17,7 @@ the above issue.
 - [ipfs filestore](#ipfs-filestore)
 - [ipfs urlstore](#ipfs-urlstore)
 - [Private Networks](#private-networks)
-- [ipfs p2p](#ipfs-p2p)
+- [ipfs p2p](#ipfs-p2p){Libp2pStreamMounting}
```

Comment:
Why isn't `{Libp2pStreamMounting}` part of the linkified area ? I belive it's better if the whole line is.

The `#*` links are generated using the `H1`, `H2`, ... tags you create by making titles with:
```markdown
## title name
```
Since you changed the name of the title down you need to change the link there too:
```suggestion
- [ipfs p2p {Libp2pStreamMounting}](#ipfs-p2p-libp2pstreammounting)
```